option.go

```diff
@@ -250,6 +250,13 @@ func OptionBreakLineCallback(fn func(*Document)) Option {
 	}
 }
 
+func OptionSetExitCheckerOnInput(fn Exitor) Option {
```

Comment:
Could you add a godoc comment?

Reply:
@c-bata  Sorry, I forgot to enable gocli and not exlude default warnings (https://github.com/golangci/golangci-lint/blame/master/README.md#L540-L545)

Reply:
How about renaming to `OptionSetExitor()`. At least I prefer to unify the term, Exitor or ExitChecker.